en/docs/get-started/community.md

```diff
@@ -0,0 +1,57 @@
+in discussions with the team and to share knowledge and challenges from your experiences, via our slack channel.
+
+* Discussions about developments: [Dev Channel](https://wso2is.slack.com/messages/dev)
+* New releases: [Release Announcement Channel](https://wso2is.slack.com/messages/releases)
+
+
+## User Forums
+
+The WSO2 Identity Server team monitors and participates in the discussions on [Stack Overflow](http://stackoverflow.com/questions/tagged/wso2is). 
+If you have any technical or programming questions related to WSO2 products, post them on Stack Overflow. Be sure to tag 
+your question with appropriate keywords so that our team can easily find your questions and provide answers. We also 
+encourage you to contribute by answering your fellow users' questions on Stack Overflow.
+
+## Report Issues
+
+We encourage you to report issues, improvements, and feature requests regarding WSO2 Identity Server through our public 
+[WSO2 Identity Server GIT Issues](https://github.com/wso2/product-is/issues).
+
+Before creating a git issue, please make sure to search all reported issues to check if someone has faced the same issue 
+or has made the same feature request.
+
+**Important**: 
+Please be advised that security issues must be reported to our private security mailing list `security@wso2.com` and not 
+as GitHub issues, to reach the proper audience. 
+Follow the [WSO2 Security Vulnerability Reporting  Guidelines](https://docs.wso2.com/display/Security/WSO2+Security+Vulnerability+Reporting+Guidelines) 
+when reporting the security issues. For more information on vulnerabilities and how they are handled, visit 
+[WSO2 Security](https://wso2.com/security).
+
+## Contribute to the Code Base
+
+WSO2 Identity Server team invites anyone (not just committers) to contribute to the code base by providing patches for 
+bug fixes or features. For this purpose, you can check out the source of the relevant GitHub repository, build the product, 
+and make changes. You can then contribute your changes by sending a pull request for review. Before sending a pull request, 
+please make sure to search the already submitted pull requests by other developers so that there won't be any duplicate 
+effort. Once the pull request is submitted, team members who are responsible for the corresponding Git repository will 
+review your proposed changes. 
+
+For more information, see [Work with the Source Code](../../extend/work-with-the-source-code/).
+
+## Contribute to the Training Materials
+
+Training materials for WSO2 Identity Server are freely available under the [CC by 4.0](https://creativecommons.org/licenses/by/4.0/) 
+license. You can check out the [WSO2 Identity Server training materials](https://github.com/wso2/WSO2-Training) and 
+submit new materials, lab exercises, and samples. To send feedback on the training content or get more information, 
+contact `training@wso2.com`.
```

Comment:
Does this appear as a link?

Reply:
added mailto link with 2cfa0a173484430435d6918a79d9f3a630a0e26f